OasisDEX/oasis-borrow
# Borrow multiply conversion issue fix

# [Borrow multiply conversion issue fix](https://app.shortcut.com/oazo-apps/story/13076/borrow-multiply-conversion-issue)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- fixed issue with converting position type while on testnet
  
## How to test 🧪
  <Please explain how to test your changes>

- position type change on ajna should work
